### ISSUE TITLE
Cross-Validation and Alternative Metrics

### ISSUE THREAD

**Issue by rodelBeronilla** [rodelBeronilla/NBA-Analysis#4]
3. Cross-Validation and Alternative Metrics
Apply cross-validation to assess model performance more robustly.
Consider using other metrics if accuracy isn't sufficient.
python
Copy code
from sklearn.model_selection import cross_val_score

# Cross-validation with F1-score as the metric
scores = cross_val_score(best_model, X_train_scaled, y_train, cv=5, scoring='f1_macro')
print("Average F1-score:", scores.mean())
